### ISSUE TITLE
Getting unknown artist when one is present

### ISSUE THREAD

**Issue by rpgdev** [RogueOneEcho/caesura#127]
I'm getting this output for this particular torrent ID. Seems like the composer label throws it off or something. Transcode folders are prefixed with Unknown artist which of course is not ideal for upload.

$ caesura verify 2549589 --verbosity trace
2025-01-14 00:10:25.717 TRCE ↩ Sending request GET /ajax.php?action=torrent&id=2549589
2025-01-14 00:10:26.551 TRCE ↩ Received response after 0.834
2025-01-14 00:10:26.574 TRCE ↩ Sending request GET /ajax.php?action=torrentgroup&id=72626
2025-01-14 00:10:26.907 TRCE ↩ Received response after 0.333
2025-01-14 00:10:26.939 DBUG * Verifying Unknown Artist - ACTUAL ALBUM NAME [ACTUAL YEAR]
2025-01-14 00:10:26.945 TRCE ↩ Found 27 flacs in: ./content/ACTUAL ARTIST NAME - ACTUAL ALBUM NAME - FLAC

Censoring ACTUAL NAMES as extra precaution.

**Comment by RogueOneEcho** [comment]
It looks like RED has no artists listed which I've never seen before. So as the artists is empty caesura defaults to 
 `Unknown Artist`.

The quickest fix would be to add all the artists to RED then caesura will just show `Various Artists`. 

- [ ] Longer term, I may change the default to `Various Artists` and log a warning to the user that no artists are listed.
